new_project/readme.md

```diff
@@ -0,0 +1,88 @@
+# Github Release Process
+
+This directory delineates the process, guidelines, and requirements for releasing 
+new projects to the [niosh-mining github page](https://github.com/niosh-mining).
+
+# Pre-Review Requirements
+
+Before starting the github release approval process, please ensure the candidate code satisfies the
+following requirements:
+
+    1. The code is stored in a git repository
```

Comment:
For some reason, the links aren't rendering correctly in this. In fact, it looks like it's interpreting this as some kind of code block? I don't see why from the markdown, though.